Documentation/Commands/Index.rst

```diff
@@ -104,6 +104,16 @@ table column for you.
 ..  code-block:: bash
     vendor/bin/typo3 make:model
 
+make:command
+============
+
+This command will create a new Command class.
+
+You will find the new file in directory ``Classes/Command/*``.
```

Comment:
Actually I would find it helpful if at the end of each command we made a prompt of which files where created or modified.